lib/std/c/darwin.zig

```diff
@@ -4155,3 +4155,15 @@ pub const vm_extmod_statistics_t = *vm_extmod_statistics;
 pub const vm_extmod_statistics_data_t = vm_extmod_statistics;
 
 pub extern "c" fn vm_stats(info: ?*anyopaque, count: *c_uint) kern_return_t;
+
+pub const _SYS_NAMELEN = 256;
+
+pub const utsname = extern struct {
+    sysname: [_SYS_NAMELEN:0]u8,
```

Comment:
These arrays are 257 bytes long (_SYS_NAMELEN 256 + sentinel 1).

Reply:
See: https://www.man7.org/linux/man-pages/man2/uname.2.html
Third paragraph in notes section
> The length of the fields in the struct varies.  Some operating
> systems or libraries use a hardcoded 9 or 33 or 65 or 257.  Other
> systems use SYS_NMLN or _SYS_NMLN or UTSLEN or _UTSNAME_LENGTH.
> Clearly, it is a bad idea to use any of these constants; just use
> sizeof(...).  Often 257 is chosen in order to have room for an
> internet hostname.

Reply:
256 is correct for macos based on https://github.com/ziglang/zig/blob/41502c6aa53a3da31b276c23c4db74db7d04796b/lib/libc/include/any-macos-any/sys/utsname.h#L72-L80